### PR TITLE
✨ Authentication with Github

### PR DESCRIPTION
**Pull request by wizzytod**
Users can sign in/sign up with Gitlab OAuth2 provider.

This PR is very similar to #303 

https://user-images.githubusercontent.com/9740575/103031610-3e123880-456f-11eb-91c9-b2d47a8bd448.mov

### CODE REVIEW

**Comment by wizzytod** [comment]
@madmath03 Please check this out if I miss something! Thanks!

File: frontend/resources/locales.json [hunk 1]
```diff
@@ -134,6 +134,15 @@
       "es" : "Entrar con Gitlab"
     }
   },
+  "auth.login-with-github-submit" : {
+    "used-in" : [ "src/app/main/ui/auth/login.cljs:154" ],
+    "translations" : {
+      "en" : "Login with Github",
+      "fr" : "Se connecter via Github",
+      "ru" : "Вход через Gitnub",
```
Comment (madmath03) [inline frontend/resources/locales.json:142]:
```suggestion
      "ru" : "Вход через Github",
```

**Review by madmath03 — CHANGES_REQUESTED** [review]
Just a small typo in Russian translation, otherwise looks very good!

**Comment by niwinz** [comment]
This is still wip?, ping me when it is ready for take a quick look and merge!
BTW nice work.

**Comment by wizzytod** [comment]
@niwinz please take a look! It is ready to merge now)

**Comment by niwinz** [comment]
Nice!, Thanks!

**Comment by niwinz** [comment]
I have merged this MR with my other branch where i working, and is pending to be merged the next days: #441

**Pull request closed without merging**